rosetta/setup.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+import setuptools
+
+package_path = os.path.join(os.path.dirname(__file__), 'rosetta')
+sys.path.append(package_path)
+from rosetta import __version__  # noqa: E402
+
+
+# Get the long description from the README file.
+with open('README.md') as fp:
+  _LONG_DESCRIPTION = fp.read()
+
+setuptools.setup(
+    name='rosetta',
+    version=__version__,
+    description='Rosetta: a Jax project for training LLM/CV/Multimodal models',
+    long_description=_LONG_DESCRIPTION,
+    long_description_content_type='text/markdown',
+    author='NVIDIA',
+    author_email='jax@nvidia.com',
+    # TODO(terry): license, url
+    packages=setuptools.find_packages(),
+    package_data={
+        '': ['**/*.gin'],  # not all subdirectories may have __init__.py.
+    },
+    scripts=[],
+    install_requires=[
+    ],
+    extras_require={
+        'test': [
+            'pytest',
+        ],
+        'lint': [
+            'ruff',
+        ],
+    },
+    # TODO(terry): classifiers
+    keywords='machinelearning multimodal llm',
```

Comment:
Some [PyPI classifiers](https://pypi.org/classifiers/) could help here.

Reply:
@sbhavani Do you have any guidance or input here w.r.t. classifiers and [keywords](https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata-keywords)?

Reply:
Here are some classifiers that seemed relevant:
* Development Status :: 2 - Pre-Alpha
* Environment :: GPU
* Environment :: GPU :: NVIDIA CUDA
* License :: OSI Approved :: Apache Software License
* Programming Language :: Python
* Programming Language :: Python :: 3
* Programming Language :: Python :: 3 :: Only
* Topic :: Scientific/Engineering :: Artificial Intelligence
* Topic :: Software Development
* Topic :: Software Development :: Libraries
* Topic :: Software Development :: Libraries :: Python Modules

As for keywords, maybe:
`machinelearning,multimodal,llm,jax-toolbox,jax`

Reply:
For keywords we could add: 
"deep learning"
For classifiers we could add:
        "Intended Audience :: Developers",
        "Intended Audience :: Education",
        "Intended Audience :: Science/Research",
        "Operating System :: OS Independent"

Reply:
Updated keywords and classifiers